### PR TITLE
Clean up ads example page.

### PR DESCRIPTION
**Pull request by lannka**
A bunch of clean ups to address #4190 :

- Made all lists of ad networks alphabetic order.
- Made the examples with script the default.
- Added links to each example at top of the page for easier navigation.
- Identified all broken examples and show them in red.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -361,7 +361,7 @@ function buildExamples(watch) {
   // Also update test-example-validation.js
   buildExample('a4a.amp.html');
   buildExample('ads.amp.html');
-  buildExample('ads.with.script.amp.html');
+  buildExample('ads-legacy.amp.html');
```
Comment (erwinmombay) [inline gulpfile.js]: maybe the ads with script should be the canonical? we do want people to explicitly add the script tag.

Comment (lannka) [inline gulpfile.js]: yes, it is. it's just under different file name now :-).

**Comment by zhouyx** [comment]
I like the clean up! 
Did we discussed this morning that we want to largely reduce the ad number in ads w/o script example?

File: gulpfile.js (hunk 1 above)
Comment (erwinmombay) [inline gulpfile.js]: lol didn't notice that. LGTM

**Comment by lannka** [comment]
I'd like to keep it as is for now, eventually it'll just go away.

**Comment by zhouyx** [comment]
SG

**Pull request merged**